tools/hmcts-snl.postman_collection.json

```diff
@@ -114,7 +170,7 @@
 			"response": []
 		},
 		{
-			"name": "delete-session",
+			"name": "refresh-day",
```

Comment:
please use the same command in the name as in the type as it's confusing (so here it would be upsert then)